*** tests/A-08-01-M-ok.out	2022-10-11 14:16:48.255088900 +0100
--- tests/A-08-01-M-ok.outhyp	2022-10-14 22:45:00.988912400 +0100
***************
*** 11,18 ****
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: BASIC|969001|cli002|IDLE|0|0
! GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
--- 11,17 ----
  2 - Registar terminal
  3 - Operações de terminal
  0 - Sair
! Escolha uma opção: GESTÃO DE TERMINAIS
  1 - Mostrar todos os terminais
  2 - Registar terminal
  3 - Operações de terminal
